README.md

```diff
@@ -344,6 +349,10 @@ And all docs are available here: [release-please docs](https://github.com/google
 
 Use [Conventional Commits](https://www.conventionalcommits.org/) to ensure that the changelogs are generated correctly.
 
+### Releasable units
+
+Release please goes through commits and tries to find "releasable units" using commit messages as guidance - it will then add these units to their respective release PR's and figures out the version number from the types: `fix` for patch, `feat` for minor, `feat!` for major. None of the other types will not be included in the changelog. So, you can use for example `chore` or `refactor` to do work that does not need to be included in the changelog and won't bump the version.
+
```

Comment:
Typo: "None of the other types will not be included in the changelog." → "None of the other types will be included in the changelog." i.e. not double negation.